packages/core/forms/src/components/forms/OIDCForm.vue

```diff
@@ -276,7 +280,7 @@ export default {
 
                   fields.push({
                     ...field,
-                    link: 'https://docs.konghq.com/hub/kong-inc/openid-connect/#jwk-record',
+                    link: this.openIdConnectLink,
```

Comment:
suggestion: should be https://developer.konghq.com/plugins/openid-connect/reference/#schema--config-client-jwk